optuna_dashboard/ts/components/GraphHistory.tsx

```diff
@@ -69,6 +69,24 @@ export const GraphHistory: FC<{
       theme.palette.mode,
       markerSize
     )
+    const element = document.getElementById(plotDomId)
+    if (element != null && studies.length >= 1) {
+      // @ts-ignore
+      element.on("plotly_click", function (data) {
+        if (data.points[0].data.mode !== "lines") {
```

Comment:
As an addition, the x coordinate of the point was used for the history redirection, which also worked for the best value line.
I want to redirect only when the trial point itself is clicked, so it will not work when the line is clicked like below.

<img width="400" alt="Screenshot 2023-09-24 at 21 56 55" src="https://github.com/optuna/optuna-dashboard/assets/23289252/1cf6bdd5-6273-45db-9d58-71040b5f6f5d">